### PR TITLE
Added terraform

### PR DESCRIPTION
**Pull request by haydenjack**
- Added terraform for the pipeline lambda function
- Added terraform for the scheduler (currently disabled on AWS to prevent duplicate RDS entries)

Potential upgrade: add CloudWatch policies to allow visualisation

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -75,3 +75,45 @@ resource "aws_scheduler_schedule" "data_management_event" {
...
+      DB_USER=var.DB_USER
+    }
+  }
+}
+
+
+# EventBridge Schedule that runs the pipeline lambda every minute
+
+resource "aws_scheduler_schedule" "pipeline-event" {
+  name                = "c9-butterflies-pipeline-schedule-tf"
```
Comment (Shurc1val) [inline terraform/main.tf:110]: I'm being petty, but any reason for this giant gap after name?

File: terraform/main.tf [hunk 2]
```diff
@@ -75,3 +75,45 @@ resource "aws_scheduler_schedule" "data_management_event" {
     role_arn = aws_iam_role.scheduler.arn 
         }
 }
+
```
Comment (david-nepo) [inline terraform/main.tf:78]: terraform syntax looks all okay

**Review by david-nepo — APPROVED** [review]
nice terrorform work

File: terraform/main.tf (hunk 1 above)
Comment (david-nepo) [reply terraform/main.tf:110]: terraform syntax allows for tabbed alignment of '=' apparently

**Pull request merged**